src/patterns/ec2-app/base.ts

```diff
@@ -11,6 +11,7 @@ import {
 } from "aws-cdk-lib/aws-cognito";
 import type { InstanceType, IPeer, ISubnet, IVpc } from "aws-cdk-lib/aws-ec2";
 import { Port } from "aws-cdk-lib/aws-ec2";
+import type { HealthCheck as ELBHealthCheck } from "aws-cdk-lib/aws-elasticloadbalancingv2";
```

Comment:
minor: ELBv2 is an Application Load Balancer, should the import alias be `ALBHealthCheck`?

Reply:
Agreed, the naming is not helpful. I think this could either be `ALBHealthCheck` or `ELBv2HealthCheck`, but not `ELBHealthCheck` which is a misleading mix of both, with overlaps with the classic load balancers (ELBs).